scheds/rust/scx_layered/src/main.rs

```diff
@@ -1296,6 +1300,11 @@ impl<'a> Scheduler<'a> {
                             mt.kind = bpf_intf::layer_match_kind_MATCH_USED_GPU_PID as i32;
                             mt.used_gpu_pid.write(*polarity);
                         }
+                        LayerMatch::AvgRuntime(min, max) => {
```

Comment:
It might be worth asserting that the `slice_us` for the layer is greater than the `min_avg_runtime_us` .